tqdm/keras.py

```diff
@@ -69,10 +69,12 @@ def __init__(self, epochs=None, data_size=None, batch_size=None, verbose=1,
     def on_train_begin(self, *_, **__):
         params = self.params.get
         auto_total = params('epochs', params('nb_epoch', None))
-        if auto_total is not None:
+        if auto_total is not None and auto_total != self.epoch_bar.total:
```

Comment:
why is this needed?

Reply:
To avoid re-assigning the total when it is provided manually by the user.